src/Agent.Listener/Capabilities/AgentCapabilitiesProvider.cs

```diff
@@ -34,6 +35,10 @@ public Task<List<Capability>> GetCapabilitiesAsync(AgentSettings settings, Cance
 #if OS_WINDOWS
             Add(capabilities, "Agent.OSVersion", GetOSVersionString());
             Add(capabilities, "Cmd", Environment.GetEnvironmentVariable("comspec"));
+
+            //This capability gets set irrespective of agent being run in auto-logon mode or not.
+            var configurationManager = HostContext.GetService<IConfigurationManager>();
```

Comment:
I can do the change but shouldnt there be a uniform model across. I see ConfigurationStore is only consumed by ConfigurationManager. At other place(s) we are calling ConfigurationManager.IsServiceConfigured() e.g., Agent class.

---
In reply to: [117336769](https://github.com/Microsoft/vsts-agent/pull/978#discussion_r117336769) [](ancestors = 117336769)